api/app_analytics/analytics_db_service.py

```diff
@@ -92,12 +95,11 @@ def get_usage_data_from_local_db(
         bucket_size=constants.ANALYTICS_READ_BUCKET_SIZE,
     )
     if project_id:
-        environment_ids = Environment.objects.filter(project_id=project_id).values_list(
-            "id", flat=True
+        environment_ids = list(  # Evaluate because analytics db has no access
+            using_database_replica(Environment.objects)
+            .filter(project_id=project_id)
+            .values_list("id", flat=True)
         )
```

Comment:
```suggestion
        # evaluate the queryset because analytics db does not have
        # access to environment/project table
        environment_ids = list( 
            using_database_replica(Environment.objects)
            .filter(project_id=project_id)
            .values_list("id", flat=True)
        )
```